plugin/src/main/kotlin/com/github/triplet/gradle/play/internal/Constants.kt

```diff
@@ -16,3 +19,22 @@ internal const val RESOURCES_OUTPUT_PATH = "generated/gpp"
 internal const val MIME_TYPE_STREAM = "application/octet-stream"
 
 internal val transport: NetHttpTransport by lazy { GoogleNetHttpTransport.newTrustedTransport() }
+
+internal fun buildCustomTransport(): NetHttpTransport? {
+    val trustStore: String? = System.getProperty("javax.net.ssl.trustStore", null)
+    val trustStorePassword: String? = System.getProperty("javax.net.ssl.trustStorePassword", null)
+
+    return if (trustStore != null) {
+        try {
+            val ks = KeyStore.getInstance(KeyStore.getDefaultType())
+            FileInputStream(trustStore).use { fis -> ks.load(fis, trustStorePassword?.toCharArray()) }
+            NetHttpTransport.Builder().trustCertificates(ks).build()
+        } catch (e: Exception){
+            // if anything goes wrong, return null
+            null
```

Comment:
I wonder if we should be a little more verbose in case of errors. Let's imagine a developer sets `customTruststore` to `true` and then makes some mistake setting up `systemProp.javax.net.ssl.trustStore` or `systemProp.javax.net.ssl.trustStorePassword`. In that case, we'd silently swallow the error and try connecting with the standard truststore (and fail). For the developer that error looks the same as if they had not set the flag in the first place. Maybe we should just pass the errors to the caller for now.